### PR TITLE
Add Signature65

### PR DESCRIPTION
**Pull request by projectkeyboard**
Adding Signature65 for Kepler FC65

## Description

Signature65 PCB for Kepler FC65 that supports ANSI 65%.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/projectkb/signature65/info.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "Kepler FC65", 
+    "url": "https://store.projectkeyboard.com", 
+    "maintainer": "projectkeyboard", 
+    "width": 16, 
+    "height": 5,
```
Comment (drashna) [inline keyboards/projectkb/signature65/info.json:6]:
```suggestion
```

File: keyboards/projectkb/signature65/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Signature65
+
+Project Keyboard ANSI only PCB for Kepler FC65. 
+
+* Keyboard Maintainer: [projectkeyboard](https://github.com/projectkeyboard)
+* Hardware Supported: STM32F072CBU6
+* Hardware Availability: [projectkeyboard](https://store.projectkeyboard.com/)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/projectkb/signature65/readme.md:9]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/projectkb/signature65/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   comment out to disable the options.
+#
+#BOOTMAGIC_ENABLE = full	# Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/projectkb/signature65/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/projectkb/signature65/signature65.c [hunk 4]
```diff
@@ -0,0 +1 @@
+#include "signature65.h"
```
Comment (drashna) [inline keyboards/projectkb/signature65/signature65.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/projectkb/signature65/readme.md (hunk 2 above)
Comment (projectkeyboard) [reply keyboards/projectkb/signature65/readme.md:9]: Added bootloader instructions.

File: keyboards/projectkb/signature65/signature65.c (hunk 4 above)
Comment (projectkeyboard) [reply keyboards/projectkb/signature65/signature65.c]: Added license header.

File: keyboards/projectkb/signature65/config.h [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0159
```
Comment (drashna) [inline keyboards/projectkb/signature65/config.h:21]: This makes me very uncomfortable.

File: keyboards/projectkb/signature65/signature65.h [hunk 6]
```diff
@@ -0,0 +1,19 @@
+#pragma once
```
Comment (drashna) [inline keyboards/projectkb/signature65/signature65.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/projectkb/signature65/rules.mk [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+BACKLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/projectkb/signature65/rules.mk:17]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**